proxy/Dockerfile

```
@@ -0,0 +1,28 @@
+FROM python:3.9.16-slim-buster
```

Now that we've bumped to python 3.10 as default

```suggestion
FROM python:3.10.12-slim-buster
```